### DISCUSSION TITLE
Blazor AOT performance vs ReadyToRun

### DISCUSSION THREAD

**Discussion by lbargaoanu** [dotnet/runtime#53994] [Q&A] [answered]
I have [this](https://github.com/lbargaoanu/RoslynInBlazor) project. It uses Roslyn to compile some code.
A lot of time is spent JIT-ing, so AOT compiling makes sense. For the console application, I get a significant benefit with ReadyToRun.
For the Blazor app compiled with AOT, I get no improvement. As far as I can tell, it is indeed compiled AOT :)

**Comment by danmoseley** [comment]
@lewing

**Comment by danroth27** [comment, marked as answer]
I moved this over to https://github.com/dotnet/runtime/issues/54265 for tracking purposes.
